firebase.json

```diff
@@ -11,6 +11,11 @@
   	  "source" : "/widgets/",
   	  "destination" : "/basic-widgets/",
   	  "type" : 301
-  	}
+  	},
+    {
+      "source" : "/tutorial/",
+      "destination" : "/introducing-flutters-widget-framework/",
```

Comment:
that's a bit of a mouthful if we're sending it around... Could we have a shorter URL?